www/index.html

```diff
@@ -22,7 +22,7 @@
     <main role="main" class="flex-shrink-0">
       <div class="container">
         <h1 class="mt-5">Firefox vs. Chrome</h1>
-        <p class="lead">From a privacy point of view.</p>
+        <p class="lead">From a privacy point of view. (default settings always)</p>
```

Comment:
Thanks for your contribution!

I'm not a native english speaker, so it's hard for me to judge what looks good (from a typographic and design point of view). Somehow I don't like brackets here. How do you feel about that?

```suggestion
        <p class="lead">From a privacy point of view. Default settings always.</p>
```

But maybe you have another suggestion (1-2 sentences).

Reply:
I like the following sentence [from the Mozilla blog](https://blog.mozilla.org/blog/2019/06/04/when-it-comes-to-privacy-default-settings-matter/):

> When it comes to privacy, default settings matter!

Maybe we can combine that. Then I like the brackets again.

```suggestion
        <p class="lead">From a privacy point of view (default settings always). When it comes to privacy, default settings matter!</p>
```